handsontable/src/plugins/contextMenu/predefinedItems/alignment.js

```diff
@@ -1,9 +1,37 @@
-import { align, getAlignmentClasses, checkSelectionConsistency, markLabelAsSelected } from '../utils';
+import {
+  align,
+  getAlignmentClasses,
+  markLabelAsSelected,
+  hasSelectionAClass
+} from '../utils';
 import { KEY as SEPARATOR } from './separator';
 import * as C from '../../../i18n/constants';
 
 export const KEY = 'alignment';
 
+/**
+ * @param {object} hot The current Handsontable instance.
+ * @param {string} htClassName The class name to check.
+ * @returns {string} The value of aria-checked parameter.
+ */
+function ariaChecked(hot, htClassName) {
+  return hasSelectionAClass(hot, htClassName).toString();
+}
```

Comment:
Is this helper needed? 🤔 WDYT about just using the `hasSelectionAClass` function within the `ariaChecked`?:
```js
ariaChecked() {
  return hasSelectionAClass(hot, 'htLeft');
},
```

Reply:
`ariaChecked` helper replaced with `hasSelectionAClass`. Unnecessary explicit `toString` removed